### ISSUE TITLE
Scan fails to start

### ISSUE THREAD

**Issue by aisalexa666** [xenesty/AkashaScanner#36]
**Describe the bug**
After pressing the "Start" button, the scanning is interrupted with the error "Interrupted"

**To Reproduce**
1. Go to the "Artifacts" or "Characters" or "Weapons" menu
2. Make sure that the fields (Level, character names, etc.) are filled in.
3. Make sure that the required tab is open in the game
4. Start scanning with the "Start" button

**Expected behavior**
The scanning process has started.

**Screenshots**
Error screenshot
![20230905T021219_1231_686_Akasha_Scanner](https://github.com/xenesty/AkashaScanner/assets/8151179/8cdaf314-17ff-47c6-ae49-46d216fd7002)

**Additional context**
1. I noticed that if I press the "Start" button several times, sooner or later the process starts and ends correctly.
2. I also noticed that the character name "Wanderer" is not saved
3. My environment and software versions:
 + Windows 10 x64
 + Microsoft Visual C+ 2015-2022 Redistributable (x64) 14.36.32532
 + I am running AkashaScanner.exe as administrator
 + I run the game bypassing the Launcher ("...\Genshin Impact game\GenshinImpact.exe")

**Comment by EnkiTWS** [comment]
I'm seeing this too. Same steps seem to get it to start, just constant re-trying of the start button. eventually it seems to go through .

**Comment by aisalexa666** [comment]
@EnkiTWS, can you tell me if you have two or more monitors?

**Comment by EnkiTWS** [comment]
> @EnkiTWS, can you tell me if you have two or more monitors?

I do. Let me guess it's because I have the scanner on a different screen than Genshin.

**Comment by aisalexa666** [comment]
Yes, thank you. That was one of my crazy guesses.

solution:
**Simply position the "AkashaScanner" program window on the same monitor as the game.**

p.s. probably an important point, but I'm too lazy to check:
The game is running full screen with the "-popupwindow" parameter.

**Comment by EnkiTWS** [comment]
Yes That was it. It needs to be on the same monitor. Which I guess that makes sense if it's looking for relative x/y screen location for something.

**Comment by tueksta** [comment]
Even on the same screen, I also have issues when using Barrier for sharing mouse/keyboard across multiple computers. So disabling this helps.
